src/libraries/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/ValidationAttribute.cs

```diff
@@ -22,12 +22,12 @@ public abstract class ValidationAttribute : Attribute
     {
         #region Member Fields
 
-        private string _errorMessage;
-        private Func<string> _errorMessageResourceAccessor;
-        private string _errorMessageResourceName;
-        private Type _errorMessageResourceType;
+        private string? _errorMessage;
+        private Func<string>? _errorMessageResourceAccessor;
```

Comment:
might make sense to just mark it as non-nullable and `= null!` to suppress warnings

Reply:
It's possible, but as it's late-initialized in at least some scenarios (via SetupResourceAccessor, where there's an actual null check) it makes a bit more sense to me to annotate it as nullable... Let me know if you feel otherwise.